examples/camera-controller/commands/delay/Commands.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "commands/common/Commands.h"
+#include "commands/delay/SleepCommand.h"
+#include "commands/delay/WaitForCommissioneeCommand.h"
+
+void registerCommandsDelay(Commands & commands, CredentialIssuerCommands * credsIssuerConfig)
+{
+    const char * clusterName      = "Delay";
```

Comment:
Why it is called "Delay"

"delay wait-for-commissionee"
"delay sleep"

not sure what the purpose of those commands

Reply:
Thank you for your comments. Basically, Chip-tool is providing Delay too. Also, It is useful when we are operating the YAML based TCs.
- Sleep: used to perform delay between operations.
- wait-for-commissionee: used to perform CASE with the device.

Reply:
Pseudo clusters available for YAML TCs: https://github.ecodesamsung.com/iot-hub/connectedhomeip/blob/test-runner/webrtc-pseudo-cluster-support/docs/testing/yaml_pseudocluster.md